### ISSUE TITLE
Accessibility (A11Y): WebView2 control is completely useless in WinUI based apps

### ISSUE THREAD

**Issue by akash07k** [microsoft/microsoft-ui-xaml#8058]
### Describe the bug

WebView2 control is completely useless in WinUI based apps.
First of all, none of the screen readers are able to recognize the content properly if we try to navigate inside the WebView2 controls, tabbing also doesn't work properly.
And most importantly, the Keystrokes such as `Alt+F4` etc. don't work.
If we are focused in a WebView2 control and try to press `Alt+F4`, then it is impossible to close the app and even we are unable to focus rest of the app controls via Shift+Tab keys.
The only solution (destructive one) is to terminate the app from the task manager.
First I thought that the issue is WebView2 specific so filed the same bug with them in April, but I checked the same with [Tauri](https://github.com/tauri-apps/tauri) which uses same WebView2 under the hood and there is no issue with it.
WebView2 work flawlessly with Tauri apps, we can properly use `Alt+F4` and all other shortcuts.
Even all the content is properly readable with all the 3 screen readers (Narrator, NVDA and JAWS)
So it is sure and evident that the issue is WinUI specific.
This is a big blocker because all of the WinUI/MAUI apps which use WebView2 under the hood are completely unusable with screen readers.
Please look into this on high priority.
Please see the issues logged with WebView2 (None of the issues are reproducing in Tauri based apps so not a WebView2 bug):
[A11Y: WebView2 control is completely inaccessible with screen readers (NVDA, JAWS and parcially ](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2330)
[A11y: The WinUI app window cannot be control using the keyboard while focus is inside a Webview2 ](https://github.com/MicrosoftEdge/WebView2Feedback/issues/2961)

### Steps to reproduce the bug

1. Create a fresh WinUI app (Stable or Preview doesn't matter). (Packaged or Unpackaged also doesn't matter).
2. Go the MainWindow.xaml
3. Write the code to include the webview in the app.
4. Now compile/run/debug the app and tab to the webview.
5. Observe navigating it with screen reader, tab, Only 1-2 controls will be announced by the screen reader.
6. Press Alt+F4 to close the app.
7. Observe, the app won't close.
8. Now try with `Tab/Shift+Tab` not possible to come back to the main App window (the native controls)

### Expected behavior

1. Whole web content should be readable with both `Tab and Arrow` keys.
2. We should be able to close the app via `Alt+F4` if we are inside WebView.
3. The `Tab/Shift+Tab` should work so that we can navigate back to the native app window controls.

### NuGet package version

None

**Comment by vagisha-nidhi195** [comment]
Adding to this, I am facing the same issue with JAWS and NVDA, both of them do not work and read out the elements. Alternatively, Narrator work well without any issue. Can you please try to prioritize this? This is a big blocker for any application.

**Comment by akash07k** [comment]
Update on this, this clearly is a WinUI bug.
I tested using WebView2 control with WinForms application too, and all is fine there.
None of the issues are present.
Every issue is only present in WinUI based apps.

**Comment by akash07k** [comment]
It is not at all stale

**Comment by akash07k** [comment]
@JesseCol @gabbybilka Can you guys please look into this issue if possible?
This is a very longstanding bug and impacting screen reader users a lot.

**Comment by JesseCol** [comment]
Adding @DmitriyKomin

**Comment by hicks86** [comment]
I am going to keep track of this ticket as this is affecting our product also and some of its users (that use screenreaders).

**Comment by omnitorsimon** [comment]
I have manage to make to make JAWS work with WebView2 and UWP. However I still have issues with ZoomText 2024 (from Freedrom Scientific) doing the same. I just does not recognize the WebView2 windows content. Trying the same page in MS Edge works fine.

**Comment by DmitriyKomin** [comment]
There are several issues being reported, addressing each one separately:

#### 1. Broken basic Narrator experience in/out of WebView2 and within the web content
> 5. Observe navigating it with screen reader, tab, Only 1-2 controls will be announced by the screen reader
> 6. Now try with Tab/Shift+Tab not possible to come back to the main App window (the native controls)

***STATUS: Not Repro***
+ I don't repro this on WinUI3 Gallery's WebView2 page + Narrator enabled (WinUI 3 Gallery v 2.2.0.0 / [WinAppSDK 1.4.231219000](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/stable-channel#version-144-14231219000)). Tabbing in/out of WebView2 works, tabbing through the web controls work (Narrator announces each control correctly).

#### 2. ALT+F4 & ALT+Space don’t work when focused on WebView2

***STATUS: Known issue in CoreWebView2***
+ Confirmed this is currently not supported in WinUI3 and, more generally, in Visual-hosted CoreWebView2 Desktop applications (such as [Win32 sample app with Visual Composition](https://learn.microsoft.com/en-us/microsoft-edge/webview2/samples/webview2samplewincomp)). 
+ Tracked by MicrosoftEdge/WebView2Feedback#2961

#### 3. WebView2 not accessible with NVDA & JAWS

***STATUS: Known issue in CoreWebView2***
+ Tracked by MicrosoftEdge/WebView2Feedback#2330

Since each of the issues is either not repro or (appropriately) tracked by a WebView2Feedback item, closing this bug.
